app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    sort_command = command.split()
+    with (open(sort_command[1], "r") as file_in,
```

Comment:
Before opening files make sure that the incoming and outcoming paths are not the same and the command name is actually `cp`